exporter/awsemfexporter/util_test.go

```diff
@@ -366,3 +366,39 @@ func TestGetLogInfo(t *testing.T) {
 	}
 
 }
+
+func TestProcessAttributes(t *testing.T) {
```

Comment:
Can you add more unit tests passing the actual reference maps used

Reply:
addressed in latest commit